### PR TITLE
docs: Updated README and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by bmar268**
Altered structure of the README file to improve readability and remove unnecessary details.

Added more details surrounding rebasing and the PR process in the CONTRIBUTING file.

Also did some general formatting and fixed grammar errors.

Relates to but doesn't complete #17

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,70 +14,92 @@ In your bug report please include the following
...
 
-All you need is Android Studio installed and set up!
+![Fork Repo button](docImages/image.png)
 
-Follow this link to download Android Studio: https://developer.android.com/studio
+From your new forked repo, simply clone the project to your machine.
 
-And watch this video to see how to set up Android Studio: https://www.youtube.com/results?sp=mAEB&search_query=how+to+set+up+android+studio
+    Copy the repo URL (click the green "Code" button and copy the link)
+    Opeining the command prompt
```
Comment (arielyxw) [inline CONTRIBUTING.md]: Here's a cheeky typo if you could correct that

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,70 +14,92 @@ In your bug report please include the following
...
+
+From your local copy of RondayView, make sure all of the changes you make are on a new branch!
+
+After you are happy with all of your changes, do a git push from your branch to github. From your forked repo go into "Pull Requests" in the navbar and click on the green "New Pull Request" button.
+
+Write a short paragraph describing what you have done in the pull request and which issue it addresses.
+
+Select the branch that you want to create the pull request for and GitHub will let you know if there are any merge conflicts. If there are merge conflicts, it will give you some suggestions on how to solve them. Once it is able to merge, you will need to wait for the build and SonarCloud analysis to pass.
+
+From there, one of out team members here at VicRoy will review your pull request and give you some constructive feedback. Once it all looks good to go, we would be honoured to merge your contribution!
```
Comment (arielyxw) [inline CONTRIBUTING.md]: Another typo here too

**Review by arielyxw — CHANGES_REQUESTED** [review]
Good work on the documentations, just a couple of typos that need to be fixed :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -14,70 +14,92 @@ In your bug report please include the following
...
+## Keeping Branches Up To Date
+
+In order to prevent your branches getting out of date, it is important to regularly pull changes from the upstream main branch. The instructions to do this via rebasing are detailed below:
+
+1) Sync the main branch on your fork with the main branch from the upstream using this button on your fork
+
+![Synce Fork Button](docImages/img.png)
 
-    Then add a /
+2) On your local main branch, then pull these changes using
```
Comment (mhardy88) [inline CONTRIBUTING.md]: I think in terms of step-by-step instructions, some people may find this sentence a little confusing. Perhaps consider changing the sentence around a bit!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -14,70 +14,92 @@ In your bug report please include the following
...
+`git pull`
 
-    Then a relevant title for your branch all in lower case with words seperated by dashes -
+3) Then rebase your feature branch with your up to date local main using
```
Comment (mhardy88) [inline CONTRIBUTING.md]: perhaps here include some details on what rebasing is and how to do it

**Review by mhardy88 — CHANGES_REQUESTED** [review]
Awesome work Ben! This is a huge improvement to our documentation, I've just added a few small nit-pick comments but overall, its great!

**Review by mhardy88 — APPROVED** [review]
Awesome work! Thank you for making those small changes, really appreciate it! More than happy to approve and merge

**Review by arielyxw — APPROVED** [review]
Awesome work, looking good for merge :)

**Pull request merged**